app/assets/javascripts/discourse/app/mixins/composer-upload-uppy.js

```diff
@@ -447,42 +448,8 @@ export default Mixin.create(ExtendableUploader, UppyS3Multipart, {
         );
       });
 
-    this._onPreProcessProgress((file) => {
-      let placeholderData = this.placeholders[file.id];
-      placeholderData.processingPlaceholder = `[${I18n.t(
-        "processing_filename",
-        {
-          filename: file.name,
-        }
-      )}]()\n`;
-
-      this.appEvents.trigger(
-        `${this.composerEventPrefix}:replace-text`,
-        placeholderData.uploadPlaceholder,
-        placeholderData.processingPlaceholder
-      );
-
-      // Safari applies user-defined replacements to text inserted programmatically.
-      // One of the most common replacements is ... -> …, so we take care of the case
-      // where that transformation has been applied to the original placeholder
-      this.appEvents.trigger(
-        `${this.composerEventPrefix}:replace-text`,
-        placeholderData.uploadPlaceholder.replace("...", "…"),
```

Comment:
I think this `replace()` could move to inside the `_uploadPlaceholder` function instead.